### PR TITLE
Rs 2022.11.20

### PR DESCRIPTION
**Pull request by rschlaikjer**
2022-11-20-172335_1920x1140_scrot

### CODE REVIEW

**Review by Petrikow — COMMENTED** [review]
fsdf

File: script/Ciel/Day 10/10_01_CIEL10_2.txt [hunk 1]
```diff
@@ -1,80 +1,97 @@
...
 -- Page 10005, Offset 27276.
 -- ホームルーム開始のチャイムと共に教室に入る。
-I enter the classroom at the same time as
+I enter the classroom just as the bell rings to signal the start of homeroom.
 }
 [sha:b84b965d84d160b7f409feb6fbc1232f71f87930]{
 -- Page 10005, Offset 27277.
 -- ……担任のノエル先生は今朝も欠勤らしい。
-It seems that Noel-sensei, who's in charge of homeroom, hasn't shown up for work this morning too.
+Noel-sensei's supposed to be in charge, but looks like she hasn't shown up today either...
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 10/10_01_CIEL10_2.txt:9]: Hmm I dunno, "in charge [of homeroom]" seems a lot more natural to me than being "assigned".

File: script/Ciel/Day 10/10_01_CIEL10_2B.txt [hunk 2]
```diff
@@ -41,37 +43,44 @@ What a joke.
...
-"――――――%{i}Aagh%{/i}!"
+"%{i}Wha%{/i}―――"
 }
 [sha:859609fab5f280673a40a581f4d0a87fa73a7147]{
 -- Page 98, Offset 27333.
 -- というより、何度見てもシエル先輩本人だった。
-No, it clearly %{i}is%{/i} Ciel-senpai, no matter how many times I check!
+// 何度見ても get outta here
+She's not just the spitting image of Ciel-senpai,
+ she %{i}is%{/i} Ciel-senpai!
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 10/10_01_CIEL10_2B.txt:83]: I feel like this one's a pretty standard pattern for this kind of gag

File: script/Ciel/Day 10/10_01_CIEL10_2B.txt [hunk 3]
```diff
@@ -101,48 +110,55 @@ She's clearly avoiding me, but I'm not in a state of mind to let that deter me.
...
-Senpai huffily turns her head away from me, with a terribly displeased expression on her face.
+Senpai huffily turns her head away from me, a terribly displeased expression on her face.
 }
 [sha:ff9eb4972f1b2d1929c6f2a4cf86e1331a86a109]{
 -- Page 102, Offset 27344.
 -- 「……すみません。気が急いてしまって、バカなコトを口走りました。席、座っていいですか？」
-"...I'm sorry. I was so on edge that I that blurted something stupid. Is it alright if I sit down here?"
+"I'm sorry...
+ I was so on edge that I just started babbling.
+ Is it alright if I sit here?"
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 10/10_01_CIEL10_2B.txt:144]: Running one's mouth has more of a shit-talking vibe, imo

File: script/Ciel/Day 10/10_01_CIEL10_2.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 10/10_01_CIEL10_2.txt:9]: Well, then at least use "in charge of our class", or something. I think it being just 'supposed to be in charge' makes it sound like something completely different.

File: script/Ciel/Day 10/10_01_CIEL10_2B.txt [hunk 4]
```diff
@@ -192,37 +215,38 @@ Without batting a eye―――
...
 -- Page 108, Offset 27358.
 -- 「ふ、ふざけないでください！
-"Please don't make me laugh!
+"D-Don't be ridiculous!
 }
 [sha:3454af935b1e53657ad3846b08d03e978ba42388]{
 -- Page 108, Offset 27359.
 -- あんなの、どう取ったって真剣な別れの言葉でしょう!?」
-That was clearly meant to be a serious farewell, no matter how one looks at it!"
+Nobody goes around saying 'farewell' to people they're going to see again!"
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 10/10_01_CIEL10_2B.txt:228]: How's this?
```diff
-Nobody goes around saying 'farewell' to people they're going to see again!"
+When you say it like %{i}that%{/i},
+ how am I supposed to interpret it as anything other than a %{i}permanent%{/i} farewell?
```
I think the permanent -> permanent can probably stay to link them here

[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 10/10_01_CIEL10_2B.txt]: Sounds good

File: script/Ciel/Day 10/10_01_CIEL10_2B.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 10/10_01_CIEL10_2B.txt:144]: That was sort of the intent, hence self-derogatory. "Sorry, I said things out of line", essentially. Well, that's just my take.

File: script/Ciel/Day 10/10_01_CIEL10_2B.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 10/10_01_CIEL10_2B.txt:83]: It still feels kinda strange without a conjunction to me. Like the lead-in is gone.

File: script/Ciel/Day 10/10_01_CIEL10_2B.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 10/10_01_CIEL10_2B.txt:144]: I see what you're getting at, but it reads like intentionally antagonistic, whereas he's just suffering verbal diarrhea

File: script/Ciel/Day 10/10_01_CIEL10_2B.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 10/10_01_CIEL10_2B.txt:83]: I meant

>In fact, she's not just...

[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 10/10_01_CIEL10_2B.txt:83]: Ok did not realize you meant at the front of the sentence. That makes way more sense.

[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 10/10_01_CIEL10_2B.txt:83]: I guess that's why I should always write an example.

**Pull request merged**